ios/FluentUI/Bottom Sheet/ContentHeightResolutionContext.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFContentHeightResolutionContext)
+public class ContentHeightResolutionContext: NSObject {
+    init(maximumHeight: CGFloat, containerTraitCollection: UITraitCollection) {
+        self.maximumHeight = maximumHeight
+        self.containerTraitCollection = containerTraitCollection
+    }
+
+    /// Maximum height of the content.
+    @objc public let maximumHeight: CGFloat
```

Comment:
Do these values need to be `@objc` and `public`? Do we expect them to be read outside Fluent, and by Objective-C clients?

Reply:
Yes to both. Clients will be doing the height resolving, and many of them are objc (including the use I'm working on).